hackclub.com.yaml

```
@@ -1642,6 +1642,15 @@ nust:
   - ttl: 1
     type: CNAME
     value: nusthackclub.netlify.app.
+ocsa:
+  - ttl: 1
+    type: TXT
+    values:
+      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+  - ttl: 300
+    type: CNAME
+    values:
+      - ghs.googlehosted.com.
```

```suggestion
ocsa:
  - ttl: 1
    type: TXT
    value:
      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
  - ttl: 300
    type: CNAME
    value:
      - ghs.googlehosted.com.
```

Whoops! That's my bad! octodns expects `value`, not `values`.